transaction.go

```diff
@@ -24,6 +24,8 @@ type Transaction struct {
 	Customer                    *Customer                 `xml:"customer,omitempty"`
 	BillingAddress              *Address                  `xml:"billing,omitempty"`
 	ShippingAddress             *Address                  `xml:"shipping,omitempty"`
+	TaxAmount                   *Decimal                  `xml:"tax-amount,omitempty"`
+	TaxExempt                   bool                      `xml:"tax-exempt,omitempty"`
```

Comment:
I might be doing something wrong, but it seems that at least as of this API version, not sending tax exempt implies that the transaction is not tax exempt, and tax exempt seems to always been sent across the wire after a transaction is created. I'm happy to push the changes I have, but if my understanding is correct, I am not sure they are an improvement for usability.